src/backtrace.c

```diff
@@ -108,8 +108,10 @@ int backtrace_snapshot(int pid, int *tids, int *index, int nr_tids)
         return -1;
 
     for (i = 0; i < snap->num_threads; ++i) {
-        printf("--------------------  thread %d (%d)  --------------------\n",
+        printf("--------------------  thread %d (%d) (",
                (index != NULL ? index[i] : i+1), snap->tids[i]);
+        print_proc_comm(snap->tids[i]);
+        printf(")  --------------------\n");
```

Comment:
Let's consider implementing better formatting. It was simple solution from the beginning but I think when changing this code it is the best time to polish output.